skimage/io/collection.py

```diff
@@ -416,39 +416,36 @@ class MultiImage(ImageCollection):
 
     Notes
     -----
-    The object that is returned can be used as a list of image-data objects,
-    where each entry in the list represents one image. In this regard it is
-    very similar to `ImageCollection`, but the two differ in the treatment
-    of multi-frame images.
+    The object that is returned is a list of image-data arrays. In this
+    regard, it is very similar to `ImageCollection`, but the two differ in
+    their treatment of multi-frame images.
 
     For a TIFF image containing N frames of size WxH, `MultiImage` stores
-    all frames of that image as a single entry of shape `(N, W, H)` in the
-    list. `ImageCollection` instead creates N entries of shape `(W, H)`.
+    all frames of that image as a single element of shape `(N, W, H)` in the
+    list. `ImageCollection` instead creates N elements of shape `(W, H)`.
 
-    For an animated GIF image, `MultiImage` in the current implementation
-    will only read one frame, while `ImageCollection` by default will read
-    all of them.
+    For an animated GIF image, `MultiImage` reads only the first frame, while
+    `ImageCollection` reads all of them by default.
 
     Examples
     --------
     >>> from skimage import data_dir
 
     >>> multipage_tiff = data_dir + '/multipage.tif'
-    >>> img = MultiImage(multipage_tiff)
-    >>> len(img) # img contains one file
+    >>> mi = MultiImage(multipage_tiff)
+    >>> len(mi)  # mi contains one element
     1
-    >>> img[0].shape # this image contains two frames of size (15, 10)
+    >>> mi[0].shape  # this element is a two-frame image of shape (2, 15, 10)
     (2, 15, 10)
 
     >>> ic = ImageCollection(multipage_tiff)
```

Comment:
I'd say the same here. How about something more descriptive?
```suggestion
    >>> image_col = ImageCollection(multipage_tiff)
```
I know lots of our documentation have `ic` around, but I'll get there eventually :slightly_smiling_face:

Reply:
I agree with your suggestions, I think they make the documentation more readable :relaxed: